README.md

```diff
@@ -21,3 +24,8 @@ WARN_TIME (default 20)
 
 Number of seconds remaining before being kicked that a player will 
 start to be warned via chat message.
+
+## Changelog:
```

Comment:
Allow folks to track what they changed to your code.

Just to give others credits for helping your mod.